Sources/ArcGISToolkit/Common/AttachmentPreview.swift

```diff
@@ -183,7 +184,6 @@ struct AttachmentPreview: View {
                     } else {
                         ProgressView()
                             .padding(8)
-                            .background(Material.thin, in: RoundedRectangle(cornerRadius: 8))
```

Comment:
I feel the appearance is a bit cleaner without this background. It's very close to the color of the rest of the unloaded tile. Thoughts? Happy to revert this otherwise.